### DISCUSSION TITLE
Unable to rename repo

### DISCUSSION THREAD

**Discussion by a84bfcae38835499** [Part-II-Computational-Physics/.github#90] [Q&A] [answered]
I had this issue with the exercises and I'm having it again with the project. I'm missing the "settings" button on the landing page of the repo.
![image](https://user-images.githubusercontent.com/125577875/227712783-a49cd852-a1cc-45d5-9e9a-00df1a39dc23.png)
Above is a picture of what I see.
![image](https://docs.github.com/assets/cb-60526/mw-1000/images/help/repository/repo-actions-settings.webp)
Above is a picture of what the github help website says I should see.

**Comment by AustenLamacraft** [comment, marked as answer]
I'm not sure why that is as I thought I set the project repos up with admin access for students. In any case it isn't too important. I've just granted you admin rights, so you should now have a "settings" tab

**Reply by a84bfcae38835499** [reply]
Thank you!
